### DISCUSSION TITLE
Outputting an annotated PDF after NER extraction

### DISCUSSION THREAD

**Discussion by mikejokic** [JohnSnowLabs/spark-nlp#13567] [Q&A] [answered]
Pipeline is standard using a pretrained Medical NLP NER model. VisualOCR does not provide the necessary entities. I can interact with the entities within the dataframe, but how can I convert it back into the original PDF but with the extracted entities overlayed? Is there a tutorial on this? 

@maziyarpanahi any help brother? Or is there a separate forum for healthcare nlp users

**Comment by maziyarpanahi** [comment, marked as answer]
This repo only supports the open-source Spark NLP.
We don't have any knowledge of any licensed products.
You can either use their repository or the slack channel dedicated to Healthcare:

https://github.com/JohnSnowLabs/johnsnowlabs
